sample/static-module/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "static-module",
+    "author": "Workleap",
+    "version": "0.0.0",
+    "description": "Static module to showcase @squide.",
+    "license": "Apache-2.0",
+    "type": "module",
+    "main": "./dist/index.js",
+    "scripts": {
+        "dev": "tsup --config ./tsup.dev.ts"
+    },
+    "peerDependencies": {
+        "react": "*",
+        "react-dom": "*",
+        "react-router-dom": "*"
+    },
+    "devDependencies": {
+        "@types/react": "18.2.0",
+        "@types/react-dom": "18.2.0",
+        "@workleap/eslint-plugin": "1.2.0",
+        "@workleap/typescript-configs": "2.1.0",
+        "react": "18.2.0",
+        "react-dom": "18.2.0",
+        "react-router": "6.11.1",
+        "react-router-dom": "6.11.1",
+        "tsup": "6.7.0"
+    },
+    "dependencies": {
+        "@squide/core": "workspace:*",
```

Comment:
squid core is not necessery since everything is reexported by squide/react-router. 

If we want client to install the squide core, it should be a peer dependency of @squide/react-router

Reply:
Good catch!

I'll keep @squide/core as a dependency rather than a peerDependency thought.